www/src/components/site-metadata.js

```diff
@@ -21,28 +21,43 @@ const SiteMetadata = ({ pathname }) => {
     }
   `)
 
-  return (
-    <Helmet defaultTitle={title} titleTemplate={`%s | ${title}`}>
-      <html lang="en" />
-      <link rel="canonical" href={`${siteUrl}${pathname}`} />
-      <meta name="docsearch:version" content="2.0" />
-      <meta
-        name="viewport"
-        content="width=device-width,initial-scale=1,shrink-to-fit=no,viewport-fit=cover"
-      />
-
-      <meta property="og:url" content={siteUrl} />
-      <meta property="og:type" content="website" />
-      <meta property="og:locale" content="en" />
-      <meta property="og:site_name" content={title} />
-      <meta property="og:image" content={`${siteUrl}${gatsbyIcon}`} />
-      <meta property="og:image:width" content="512" />
-      <meta property="og:image:height" content="512" />
+  const {
+    metaImage = `${siteUrl}${gatsbyIcon}`,
+    metaTitle = title,
+    metaDescription = `${siteUrl}`,
```

Comment:
Probably something better we can use here. Seems weird for the description to be the siteUrl.

Reply:
I think I had something using the title instead of the siteUrl, but it was suggested by @fk to maybe use the siteUrl. I can change it to whatever!